tools/cli/commands/build.js

```diff
@@ -831,10 +857,33 @@ async function buildProject( t ) {
 		await fs.writeFile( `${ buildDir }/.gitattributes`, rules, { encoding: 'utf8' } );
 	}
 
+	// Get the proeject version number from the changelog.md file.
+	let projectVersionNumber = '';
+	try {
+		const changelogText = await fs.readFile( `${ t.cwd }/CHANGELOG.md`, { encoding: 'utf8' } );
```

Comment:
We should be able do this without having to read the entire file into memory and then splitting it into a huge array. You might look at [readline](https://nodejs.org/api/readline.html#example-read-file-stream-line-by-line) to read line by line until we find the first header.

Reply:
acd011c